gradle/libs.versions.toml

```diff
@@ -47,4 +47,4 @@ rxrelay = "com.jakewharton.rxrelay2:rxrelay:2.1.1"
 
 junit-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "junit5" }
 junit-params = { module = "org.junit.jupiter:junit-jupiter-params", version.ref = "junit5" }
-junit-engine = { module = "org.junit.jupiter:junit-jupiter-engine", version.ref = "junit5" }
+junit-engine = { module = "org.junit.jupiter:junit-jupiter-engine", version.ref = "junit5" }
```

Comment:
can you revert this or re-introduce the new line?

Reply:
sorry, fixed.